### DISCUSSION TITLE
How can I use timestamped with Subtitle Edit?

### DISCUSSION THREAD

**Discussion by aaron2221** [linto-ai/whisper-timestamped#88] [General]
I'm a newbie and getting this to run without Subtitle Edit looks very complicated to me..

**Comment by Jeronymous** [comment]
What do you mean by "run without Subtitle Edit"?
Can you please describe your usage and where you block?

**Reply by aaron2221** [reply]
If you could make Whisper TS in a standalone version like "Faster Whisper" did here: https://github.com/Purfview/whisper-standalone-win it would be great, because this way I'll be able to run it from Subtitle Edit instead of using the complicated (for me) command line.

**Comment by darnn** [comment]
Subtitle Edit has automated downloads and a basic GUI for various implementations of Whisper (Whisper-TS not being one of them, at the moment), I believe they just mean it's hard to follow the instructions of installing Whisper-TS manually and using the command line.

**Reply by aaron2221** [reply]
Yes! That's it, thank you :) the command line is really confusing me and I'll have to install python and other stuff... Subtitle Edit makes it so much easier and also you get to edit the subtitles and make them perfect there.

**Comment by darnn** [comment]
You can check out the way it works there (Video->Audio to text (Whisper), if you have any interest in providing a Windows build that could just be slotted in there:
https://github.com/SubtitleEdit/subtitleedit/releases/
